cmd/swarmctl/secrets/create.go

```diff
@@ -0,0 +1,44 @@
+package secrets
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/cmd/swarmctl/common"
+	"github.com/spf13/cobra"
+)
+
+var createCmd = &cobra.Command{
+	Use:   "create <secret name>",
+	Short: "Create a secret",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return errors.New("create command takes a unique secret name as an argument, and accepts secret data via stdin")
```

Comment:
I guess the UX doesn't really matter for `swarmctl` as users will be using the docker CLI, but how to you envision that UX?

As in, do we expect secrets to be passed through `stdin` only?

I have a counter proposal (assuming `foo` is the secret name and `bar` the secret value):
```
Pass as an argument:
docker secret create foo bar

Pass to stdin:
echo bar | docker secret create foo -

Pass as a file:
echo bar > value.txt && docker secret create foo -f value.txt
```

Thoughts? /cc @cyli @diogomonica @aaronlehmann @stevvooe